*** setup.py	2014-02-17 15:03:49.000000000 +0100
--- setup_no_tests.py	2014-05-25 11:47:39.616477862 +0200
***************
*** 16,22 ****
        description='Code Metrics in Python',
        platforms='any',
        long_description=readme,
!       packages=find_packages(),
        tests_require=['tox'],
        install_requires=['mando', 'colorama', 'pathfinder'],
        entry_points={'console_scripts': ['radon = radon:main']},
--- 16,22 ----
        description='Code Metrics in Python',
        platforms='any',
        long_description=readme,
!       packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
        tests_require=['tox'],
        install_requires=['mando', 'colorama', 'pathfinder'],
        entry_points={'console_scripts': ['radon = radon:main']},
