*** setup.py	2014-04-14 19:42:32.319448063 +0200
--- setup_no_test.py	2014-04-14 19:43:04.592999162 +0200
***************
*** 40,46 ****
          'Topic :: Utilities',
      ],
      platforms='any',
!     packages=find_packages(),
      include_package_data=True,
      install_requires=reqs,
      test_suite='nose.collector',
--- 40,46 ----
          'Topic :: Utilities',
      ],
      platforms='any',
!     packages=find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"]),
      include_package_data=True,
      install_requires=reqs,
      test_suite='nose.collector',
